fastchat/model/model_adapter.py

```diff
@@ -5,15 +5,20 @@
 from typing import List, Optional
 import warnings
 
-from fastchat.modules.gptq import GptqConfig, load_gptq_quantized
-
 if sys.version_info >= (3, 9):
     from functools import cache
 else:
     from functools import lru_cache as cache
 
 import psutil
 import torch
+
+# Try to import intel_extension_for_pytorch, only required for "xpu" device
+try:
+    import intel_extension_for_pytorch as ipex
```

Comment:
Move this import under
```elif device == "xpu":```
and raise a warning if not found.

Reply:
Yeah, good idea!